### ISSUE TITLE
Hex logo

### ISSUE THREAD

**Issue by Nowosad** [mtennekes/tmap#409]
Hi @mtennekes , maybe you would be interested in adding a hex logo to **tmap**? I have prepared three versions of the logo below. Let me know what you think about them.

![hex-tmap1](https://user-images.githubusercontent.com/3457131/78889069-479c5480-7a63-11ea-9d76-e8a0acc55c3f.png)
![hex-tmap3](https://user-images.githubusercontent.com/3457131/78889077-49661800-7a63-11ea-95b6-31db1ce6815c.png)
![hex-tmap2](https://user-images.githubusercontent.com/3457131/78889080-49661800-7a63-11ea-9442-d8c2eecd4a3f.png)

**Comment by mtennekes** [comment]
Excellent idea, Jakub! Love them. My ranking of those three would be 1,3,2, especially because of how the letters stand out.

I'll leave this issue open for a bit to see whether there is more inspiration coming (since the creation process is fun).

**Comment by rvalavi** [comment]
@Nowosad beautiful hex logos! The first one 👍

**Comment by statnmap** [comment]
What would you think about something lighter like the following one ?
I was inspired by your propositions, the choice of the colors that is characteristic of {tmap}.  
I took some liberty with the projection too... :smile: 
![propal_tmap](https://user-images.githubusercontent.com/21193866/79363496-429a4200-7f48-11ea-865f-6df3b9a90b1d.png)

**Comment by mtennekes** [comment]
That's beautiful @statnmap ! I love the simplifications (although I also love the high-res logo's from @Nowosad )

I could easily settle for one of the suggestions. However, in the perfect world, I would like to add something that represent a certain statistic/theme. tmap isn't just a package for nicely looking basemaps, but also and especially for "thematic" maps.

**Comment by statnmap** [comment]
Do you have a real map that you think represents well the {tmap} package ?

**Comment by mtennekes** [comment]
Not really. The examples in the JSS papers (also posted on https://github.com/mtennekes/tmap) could be used, but there may be nicer examples.

**Comment by statnmap** [comment]
And something like this ? This is a little supercharged to me, but it's an idea
![tmap_full](https://user-images.githubusercontent.com/21193866/79953635-78758400-847c-11ea-81ff-c4e83735b0c2.png)

**Comment by mtennekes** [comment]
@Nowosad and @statnmap : Did you make the logos with R and/or another tool? Could you share the source code/files?

**Comment by Nowosad** [comment]
My approach: first draft with https://connect.thinkr.fr/hexmake/ and the final versions with GIMP. I can share my GIMP files. I also plan to make some new versions, but not sooner than late May.

**Comment by statnmap** [comment]
I'll tell Colin that you used his Shiny App, I'll be happy :wink:   
On my side, I use Inkscape to allow for vectorised images. So that I can create any size with a proper resolution. I could also share the source file.

**Comment by mtennekes** [comment]
Great! Looking forward to other propositions! I'll also give it a go when time allows.



From my side there are a few requirements, also considering other R package hex logos:
![image](https://user-images.githubusercontent.com/2444081/80571678-54c2b880-89fd-11ea-9243-a9f64acbedcd.png)

* The logo should stand out in small size, with the text "tmap" clearly readable.
* Often "less is more".
* I love green. But apparently you already sensed this!
* It should highlight the purpose of `tmap` (i.e. thematic maps)
* Ideally, it should have something funny/outside the box (like the testthat icon)

@statnmap Which app is this?

**Comment by statnmap** [comment]
Here is the Inkscape project, inside the zip file. We will the evolution of my ideas...  
If you want to try some things.
[tmap.zip](https://github.com/mtennekes/tmap/files/4551844/tmap.zip)

_The Shiny app : https://connect.thinkr.fr/hexmake/_

**Comment by mtennekes** [comment]
<img src="https://user-images.githubusercontent.com/2444081/106523217-12c12880-64e1-11eb-8d55-01442d535400.png" width="200">
